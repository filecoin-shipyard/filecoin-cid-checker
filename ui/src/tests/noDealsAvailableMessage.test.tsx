import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'

import { NoDealsAvailableMessage } from '../components/noDealsAvailableMessage.component'

describe('<NoDealsAvailableMessage /> spec', () => {
  it('renders the component', async () => {
    const { container } = render(<NoDealsAvailableMessage />)
    expect(container).toMatchSnapshot()
  })
})
