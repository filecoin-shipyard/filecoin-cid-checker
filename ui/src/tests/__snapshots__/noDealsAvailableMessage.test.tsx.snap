// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoDealsAvailableMessage /> spec renders the component 1`] = `
<div>
  <div
    class="sc-bdnylx dAbJmV"
  >
    No Deals available
  </div>
</div>
`;
